python/ray/tests/kuberay/test_autoscaling_e2e.py

```diff
@@ -366,6 +368,14 @@ def testAutoscaling(self):
 
 if __name__ == "__main__":
     import pytest
-    import sys
+
+    kubeconfig_base64 = os.environ.get("KUBECONFIG_BASE64")
+    if kubeconfig_base64:
```

Comment:
the most "interesting" thing is that I need to send in this file via an env var. this is because the cluster is setup outside the container, but needs to be connected inside the container, and this is the easiest way I can think of to send in this file.

the file's content contains a generated certificate for connecting the kind cluster.

Reply:
> this is because the cluster is setup outside the container

Do you mean the Kind Kubernetes cluster is set up outside the Buildkite runner container?

Reply:
the test is running in another container via @can-anyscale 's `ray_ci` wrapper, not the buildkite runner container.

the kind cluster is setup inside the buildkite runner container, and I need to send in the kubeconfig into the container that runs the test.

Reply:
and the cloned repo has to be on a dir called `rayci` (like before it has to be `ray`)

we can refactor the test to be easier to run over time, but this test is probably not high priority at the moment.